router/router.go

```diff
@@ -228,12 +230,10 @@ func New(cfg *config.Configuration, rateConvertor *currencies.RateConverter) (r
 	}
 
 	p, _ := filepath.Abs(infoDirectory)
-	bidderInfos := adapters.ParseBidderInfos(cfg.Adapters, p, openrtb_ext.BidderList())
+	bidderInfos := adapters.ParseBidderInfos(cfg.Adapters, p, openrtb_ext.BidderNames())
 
-	disabledBidders := map[string]string{
-		"indexExchange": "Bidder \"indexExchange\" has been deprecated and is no longer available. Please use bidder \"ix\" and note that the bidder params have changed.",
-	}
-	activeBiddersMap := exchange.DisableBidders(bidderInfos, disabledBidders)
+	activeBidders := exchange.GetActiveBidders(bidderInfos)
+	disabledBidders := exchange.GetDisabledBiddersErrorMessages(bidderInfos)
```

Comment:
`DisableBidders` is separated, since it's doing two entirely separate things which just happen to operator over the same data. We don't care about performance here since this is a run once setup when the server begins.